CommandLine/Parser.cs

```diff
@@ -16,10 +16,18 @@ public Parser(params Option[] options) : this(new ParserConfiguration(options))
         {
         }
 
+        public Parser(bool unbundle, params Option[] options) : this(new ParserConfiguration(options, allowUnbundling: unbundle))
```

Comment:
Since we could foresee an explosion of constructor variants with this pattern, my plan had been to remove the constructors that set individual settings (such as the following on with a `delimiters` parameter), and only support the `ParserConfiguration` constructor.